hub-detect/src/main/resources/hub-detect-sh

```diff
@@ -57,63 +66,53 @@ get_detect() {
     CURRENT_VERSION=$( <$VERSION_FILE_DESTINATION )
   fi
 
-  curl $DETECT_CURL_OPTS -o $VERSION_FILE_DESTINATION https://blackducksoftware.github.io/hub-detect/latest-commit-id.txt
-  LATEST_VERSION=$( <$VERSION_FILE_DESTINATION )
+  curl -sSL $DETECT_CURL_OPTS -o "${VERSION_FILE_DESTINATION}" https://blackducksoftware.github.io/hub-detect/latest-commit-id.txt
+  LATEST_VERSION=$(<"${VERSION_FILE_DESTINATION}")
 
   if [ $DETECT_USE_SNAPSHOT -eq 1 ]; then
     if [ -z "${DETECT_RELEASE_VERSION}" ]; then
-      echo "will look for snapshot: hub-detect-latest-SNAPSHOT.jar"
       DETECT_SOURCE="https://test-repo.blackducksoftware.com/artifactory/bds-integrations-snapshot/com/blackducksoftware/integration/hub-detect/latest-SNAPSHOT/hub-detect-latest-SNAPSHOT.jar"
       DETECT_DESTINATION="${DETECT_JAR_PATH}/hub-detect-latest-SNAPSHOT.jar"
+      echo "will look for snapshot: ${DETECT_SOURCE}" >&2
     else
       DETECT_SOURCE="https://test-repo.blackducksoftware.com/artifactory/bds-integrations-snapshot/com/blackducksoftware/integration/hub-detect/${DETECT_RELEASE_VERSION}/hub-detect-${DETECT_RELEASE_VERSION}.jar"
       DETECT_DESTINATION="${DETECT_JAR_PATH}/hub-detect-${DETECT_RELEASE_VERSION}.jar"
+      echo "will look for release: ${DETECT_SOURCE}" >&2
     fi
   else
     if [ -z "${DETECT_RELEASE_VERSION}" ]; then
-      DETECT_RELEASE_VERSION=$(curl $DETECT_CURL_OPTS 'https://test-repo.blackducksoftware.com/artifactory/api/search/latestVersion?g=com.blackducksoftware.integration&a=hub-detect&repos=bds-integrations-release')
+      DETECT_RELEASE_VERSION=$(curl -sSL $DETECT_CURL_OPTS 'https://test-repo.blackducksoftware.com/artifactory/api/search/latestVersion?g=com.blackducksoftware.integration&a=hub-detect&repos=bds-integrations-release')
```

Comment:
looking up the Curl man page, I think this is supposed to be -ssl not -sSL. Could you change these?

Reply:
Each letter in the -XYZ notation is a separate option.  I added the lower-case "s" to hide the progress screen and an upper-case S to show error messages such as Zscaler certificate errors.

My employer has a ticket 00085307 on specifying a space-separated value in one of the parameters,

    --detect.maven.build.command="compile -s settings.xml"